src/js/components/Articles/ArticleReader.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from "react";
+import Navbar from "../Navbar/Navbar";
+import { createGlobalStyle } from "styled-components";
+import { ReactReader } from "./modules";
+import { NavLink } from "react-router-dom";
+import "./ArticleButtons.scss";
+import Share from "./Share";
+import "../Fragments/Navbuttons.scss";
+import Bookmark from "react-bookmark";
+//import Bookmark from "./Bookmarks";
+
+import {
+  Container,
+  ReaderContainer,
+  FontSizeButton,
+  ExitFullScreenButton
+} from "./Components";
+
+const storage = global.localStorage || null;
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
put this in a .css .scss(some styling file) class pls and then inport it

Reply:
resolved